arch/arm64/configs/bcmrpi3_defconfig

```diff
@@ -708,6 +708,7 @@ CONFIG_STMPE_SPI=y
 CONFIG_MFD_ARIZONA_I2C=m
 CONFIG_MFD_ARIZONA_SPI=m
 CONFIG_MFD_WM5102=y
+CONFIG_REGULATOR_RASPBERRYPI_TOUCHSCREEN_ATTINY=m
```

Comment:
This setting isn't working - it needs at least CONFIG_REGULATOR=y, but that doesn't seem to be sufficient.